src/main/java/BottleOpener/Tasklist.java

```diff
@@ -0,0 +1,105 @@
+package BottleOpener;
+
+import java.util.ArrayList;
+
+/**
+ * The {@code BottleOpener.Tasklist} class manages a list of tasks, allowing operations such as adding,
+ * deleting, marking, and unmarking tasks. It provides functionality to interact with the
+ * task list and retrieve information about the tasks.
+ */
+public class Tasklist {
+    public int index;
+    public ArrayList<Task> tasklist;
```

Comment:
Perhaps consider renaming it to taskLists, following camelCasing as well as distinguishing clearly between single-valued and multi-valued variables! Small nit picking!